spec/02-integration/07-sdk/06-worker_events_spec.lua

```diff
@@ -0,0 +1,156 @@
+local helpers = require "spec.helpers"
+
+describe("worker_events, ", function()
+  local strategy = "off"
+  local test_cases = {"string", "table", }
+  local payload_size = 70 * 1024
+  local max_payloads = { 60*1024, 130*1024 }
+  local business_port
+
+  local fixtures = {
+    http_mock = {
+      worker_events = [[
+        server {
+          server_name example.com;
+          listen %s;
+
+          location = /payload_string {
+            content_by_lua_block {
+              local PAYLOAD_TOO_BIG_ERR = "failed to publish event: payload too big"
+              local SOURCE, EVENT       = "foo", "string"
+              local worker_events       = kong.worker_events
+              local cjson               = require "cjson.safe"
+              local payload_received
+
+              local function generate_data()
+                return string.rep("X", %s)
```

Comment:
should be `%d`?

Reply:
it seems not matter here, we offer a string which actually presents an integer.